EIPS/eip-7923.md

```diff
@@ -169,6 +176,46 @@ index bf34fbee..f63b7460 100644
 
      def memory_write(self, start_position: int, size: int, value: bytes) -> None:
          return self._memory.write(start_position, size, value)
+diff --git a/eth/vm/forks/frontier/computation.py b/eth/vm/forks/frontier/computation.py
+index 51666ae0..443f82b5 100644
+--- a/eth/vm/forks/frontier/computation.py
++++ b/eth/vm/forks/frontier/computation.py
+@@ -29,6 +29,7 @@ from eth.exceptions import (
+     InsufficientFunds,
+     OutOfGas,
+     StackDepthLimit,
++    VMError,
+ )
+ from eth.vm.computation import (
+     BaseComputation,
+@@ -87,12 +88,21 @@ class FrontierComputation(BaseComputation):
```

Comment:
This looks fine, I am not aware of the entire codebase though but does this only apply to Frontier (chainstart)?. Should not write this for all forks but maybe a comment/note that this accounting of the pages should be done for all forks if these kind of `Computation` methods exist for each fork.

Reply:
all of the `Computation`s inherit from the previous fork's computation class.. since it's a reference implementation, i don't feel the need to add full fork choice rules. i can add a note though